src/helpers/resolveCardForLevel.js

```diff
@@ -11,15 +11,10 @@ export default card => {
     return null
   }
 
-  // In case of a token card, the ID is the card token ID + a unique identifier
-  // in order to differentiate similar card tokens. To be able to display the
-  // card correctly despite have a unique ID which obviously doesn’t exist in
-  // the card database, we define a `displayId` which is used to know which card
-  // to render, regardless of the actual ID.
-  const displayId = card.token ? card.id.split(':')[0] : id
+  const displayId = id.split('_')[0]
```

Comment:
```suggestion
  const [displayId] = id.split('_')
```
… if you prefer. You don’t have to. :)